### PR TITLE
Cleanup artifact names

### PR DESCRIPTION
**Pull request by lulf**
Using image-specific names for each image artifact, required for CPaaS. 
Remove module-template.yaml as it is no longer needed
Use empty md5 to avoid stale md5 being used for manual image

### CODE REVIEW

File: iot-auth-service/image.yaml [hunk 1]
```diff
@@ -52,7 +52,7 @@ run:
       user: 185
       cmd:
           - "/bin/launch_java.sh"
-          - "/opt/iot-component.jar"
```
Comment (ctron) [inline iot-auth-service/image.yaml:55]: All IoT components are named `iot-component.jar` since that is used by the `hono.*` modules.

**Review by ctron — CHANGES_REQUESTED** [review]

File: iot-auth-service/image.yaml (hunk 1 above)
Comment (lulf) [reply iot-auth-service/image.yaml:55]: Yes, i need to make the module install per image

**Comment by ctron** [comment]
Isn't there a chance that you have two modules each. e.g. for the license part, one common module, which adds the `license.sh`, and a second (local) module, which calls the `license.sh` script. That way the code duplication would be a little less.

**Comment by lulf** [comment]
@ctron Its not possible at present, but there will be a possibility in future cekit versions to pass parameters to modules (and effectively scripts it executes), which should allow us to reduce the duplication. I would like to merge this for now, so that we can get cpaas running from the main branch.

**Review by ctron — APPROVED** [review]

**Pull request merged**